### PR TITLE
update set image command in a simpler way

### PR DESCRIPTION
**Pull request by cloudmelon**
update set image command in a simpler way

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by cloudmelon** [comment]
@jaredbhatti please review my changes thanks :)

Best wishes from the UK
Melony

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -158,6 +158,10 @@ Follow the steps given below to update your Deployment:
     ```shell
     kubectl --record deployment.apps/nginx-deployment set image deployment.v1.apps/nginx-deployment nginx=nginx:1.9.1
     ```
+    or simply use the following command : 
+    ```shell
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/controllers/deployment.md]: pls pay attention to the indentation, the current indentation may not work.

Comment (cloudmelon) [reply content/en/docs/concepts/workloads/controllers/deployment.md]: please @tengqm  review my changes after fixing the indentation, thanks :)

**Pull request merged**